aten/src/ATen/function_wrapper.py

```diff
@@ -997,18 +1055,23 @@ def process_native(option):
         assert option['python_module'] == '' or option['python_module'] == 'nn', \
             "Found python_module of {} for decl {}, but only \'\' string or \'nn\' are supported".format(
                 option['python_module'], option['name'])
-        formals = native_get_formals(option)
+        formals = native_get_formals(option, False)
+        schema_order_formals = native_get_formals(option, True)
         option['formals_list'] = formals
         option['formals'] = [format_formal(f) for f in formals]
         option['formals_with_defaults'] = [formal_with_default(f) for f in formals]
         option['returns'] = native_get_return_types(option)
         option['return_type'] = format_return_type(option['returns'])
         option['return_call'] = 'return ' if option['return_type'] != 'void' else ''
         option['actuals'] = [f['name'] for f in formals]
+        option['schema_order_actuals'] = [f['name'] for f in schema_order_formals]
 
         option['formals_types'] = [f['type'] for f in option['formals_list']]
 
         option['cpp_signature'] = "{} ({})".format(option['return_type'], ", ".join(option['formals_types']))
+        option['schema_order_cpp_signature'] = option['cpp_signature']\
+            .replace('const TensorOptions &',
```

Comment:
This looks hella fragile

Reply:
how would you do it instead?

Reply:
Well, typically, you would have done this at the time `cpp_signature` was generated, where you have more structure than just a string that you're regexing over (as you are here.) This might be inconvenient given the code, but it's what I would have expected.

Reply:
+1, how hard would it be to do this at `cpp_signature` creation time? I guess it's not a dealbreaker but this is kind of an antipattern in codegen (important logic done by downstream mutating declarative structures based on string matching)

Reply:
fixed